app/Bus/Commands/ComponentGroup/UpdateComponentGroupCommand.php

```diff
@@ -49,31 +49,49 @@
     public $collapsed;
 
     /**
+     * To whom should the component group be visible?
+     *
+     * @var int
+     */
+    public $visible;
+
+    /**
+     * The id of the creator of the component group.
+     */
+    public $created_by;
+
+    /**
      * The validation rules.
      *
      * @var string[]
      */
     public $rules = [
-        'name'      => 'string',
-        'order'     => 'int',
-        'collapsed' => 'int|between:0,3',
+        'name'       => 'string',
+        'order'      => 'int',
+        'collapsed'  => 'int|between:0,3',
+        'visible'    => 'int|between:0,3',
+        'created_by' => 'int',
     ];
 
     /**
      * Create a add component group command instance.
      *
-     * @param \CachetHQ\Cachet\Models\ComponentGroup $group
-     * @param string                                 $name
-     * @param int                                    $order
-     * @param int                                    $collapsed
+     * @param ComponentGroup $group
```

Comment:
Will update. But, since it was already imported, I didn't see the point in using the FQNS.